edx_proctoring/templates/proctored_exam/other_exam_in_progress.html

```diff
@@ -0,0 +1,15 @@
+{% load i18n %}
+<div class="failure sequence proctored-exam" data-exam-id="{{exam_id}}">
+  <h3>
+    {% blocktrans %}
+      Complete in-progress exam
+    {% endblocktrans %}
+  </h3>
+
+  <p>
+    {% blocktrans %}
+      You must complete your currently in-progress exam before you can attempt this one.
```

Comment:
Since we already have the running attempt available, we should be able to link back to the other exam, right?